skimage/data/_registry.py

```diff
@@ -116,6 +116,8 @@
     "data/bw_text_skeleton.npy": "9ff4fc23c6a01497d7987f14e3a97cbcc39cce54b2b3b7ee33b84c1b661d0ae1",
     "data/_blobs_3d_fiji_skeleton.tif": "5182a2a94f240528985b8d15ec2aebbd5ca3c6b9214beff1eb6099c431e12b7b",
     "data/checker_bilevel.png": "2e207e486545874a2a3e69ba653b28fdef923157be9017559540e65d1bcb8e28",
+    "restoration/tests/astronaut_rl.npy":
```

Comment:
would it be possible to host this image on https://gitlab.com/scikit-image/data/ in order to limit the size of the package? See for example what has been done in https://github.com/scikit-image/scikit-image/pull/4809/

Reply:
Is this really wise?  It feels inefficient, and trying to figure out how this will proceed seems painful:

- This requires me to setup an account in entirely different service.
- How do I run the tests prior to the data file appearing in gitlab?
- Back and forth on this PR could require additional PRs on gitlab to update the testdata.

This feels like a lot of extra effort for contributors, and will likely discourage contributions.

I will attempt it.

Reply:
@chrisroat you make good points! However, since we usually squash-and-merge at the end of a PR, you can leave the hosting change until the very last commit. We should definitely have better documentation and process around this, thank you for raising these issues — and for giving it a go!

Reply:
Also, just btw, we are having discussions about this in #4538. The *first* motivation for gitlab is that it *seemed* we were hitting github bandwidth limits with large data files being downloaded from github. But we never really verified that.

Reply:
are tests not part of the sdist package? My initial remark was because I cared about the package size, for the repo size as you're pointing out @jni we're not so sure.

Reply:
ok, we do ship the test files also with the pip package, so that's the reason why we don't want to include too many files.

Reply:
@emmanuelle but new test files can/should be managed by pooch and not shipped. We can certainly put them in this repo — as I said part of it was a hunch to avoid hitting the github bandwidth limits. All of this can be revisited easily, since it is a matter of updating some URLs.

> for the repo size as you're pointing out @jni we're not so sure.

Since some data files (e.g. cells.tif) are 30MB+, it would be pretty easy to get to a multi-100MB repo download, which is not ideal! So I still think having new data in a separate repo is a worthwhile goal.

Reply:
we should document this process in the dev doc.